### DISCUSSION TITLE
v1.2.0-preview1

### DISCUSSION THREAD

**Discussion by btueffers** [microsoft/WindowsAppSDK#2942] [Announcements]
### 1.2 Preview 1 Release of Windows App SDK 

The first preview of WinAppSDK 1.2 is now available! To see everything that's new and changed, see the full [release notes here](https://aka.ms/windowsappsdk/1.2/1.2-preview1/release-notes). 

### Updating to this version
The 1.1-preview2 package is available [here](https://aka.ms/windowsappsdk/1.2/1.2-preview1).

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue here, on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.2.0-preview1'>v1.2.0-preview1</a>.</em>

**Comment by kmgallahan** [comment]
Looking through the new notifications API, am I correct that there is no support for scheduling / unscheduling notifications at this time?

The CommunityToolkit Notifications package had this feature. Not super critical for my needs right now though.

**Comment by kmgallahan** [comment]
Also you have a duplicate discussion here: https://github.com/microsoft/WindowsAppSDK/discussions/2941

And a copy+paste typo:

> The 1.1-preview2 package is available [here](https://aka.ms/windowsappsdk/1.2/1.2-preview1).

**Comment by jeremy-visionaid** [comment]
Rendering performance is still borked for high refresh screens :(

**Comment by lighterhy** [comment]
So. Err. What's in the roadmap? I can't find an upcoming feature list.
